test/sol6/kyberFeeHandler.js

```diff
@@ -1631,11 +1633,31 @@ contract('KyberFeeHandler', function(accounts) {
             });
         });
     });
+
+    it("test upgradeable - update network address", async() => {
+        admin = accounts[9];
+        tempNetwork = await KyberNetworkProxy.new(admin);
+        tempProxy = await KyberNetworkProxy.new(admin);
+        tempFeeHandler = await FeeHandler.new(daoSetter, tempProxy.address, kyberNetwork, knc.address, BURN_BLOCK_INTERVAL, burnConfigSetter);
```

Comment:
`tempNetwork` should be Network instance (or just normall addr to test)?

Reply:
fixed. change to network instance